modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -572,13 +576,43 @@ public static Preferences getPreferences() {
      *         <tr><td>{@code GTK.warning_color}</td><td>{@link Color}</td></tr>
      *         <tr><td>{@code GTK.error_color}</td><td>{@link Color}</td></tr>
      *         <tr><td>{@code GTK.success_color}</td><td>{@link Color}</td></tr>
+     *         <tr><td>{@code GTK.enable_animations}</td><td>{@link Boolean}</td></tr>
      *         <tr></tr>
      *     </tbody>
      * </table>
      *
      * @since 22
      */
-    public interface Preferences extends ObservableMap<String, Object> {
+    public sealed interface Preferences extends ObservableMap<String, Object>
```

Comment:
is there any way we can convert this to a regular class here, so it can be inspected?

what's the point of declaring an interface just to allow for one possible implementation?

Reply:
No, that would be a mistake.

Reply:
please ignore.  Using reflection and/or BeanInfo on Platform.Properties.class worked.